packages/ui-top-nav-bar/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "private": true,
+  "name": "@instructure/ui-top-nav-bar",
+  "version": "8.27.0",
+  "description": "A UI component library made by Instructure Inc.",
```

Comment:
This should be a more specific description. Something like "A responsive top navigation bar with light and dark theme"

Reply:
Ofc, this is just the skeleton for the folder placeholder, I'll do the rest on the feature branch :)